src/lib/utils.ts

```diff
@@ -1,29 +1,17 @@
", "tc", "tb", "ta", "sz", "sy", "sx", "sw", "sv", "su", "st", "sr", "sq", "sp", "so", "sn", "sm", "sl", "sk", "sj", "si", "sh", "sg", "sf", "se", "sd", "sc", "sb", "sa", "rz", "ry", "rx", "rw", "rv", "ru", "rt", "rs", "rq", "rp", "ro", "rn", "rm", "rl", "rk", "rj", "ri", "rh", "rg", "rf", "re", "rd", "rc", "rb", "ra", "qz", "qy", "qx", "qw", "qv", "qu", "qt", "qs", "qr", "qp", "qo", "qn", "qm", "ql", "qk", "qj", "qi", "qh", "qg", "qf", "qe", "qd", "qc", "qb", "qa", "pz", "py", "px", "pw", "pv", "pu", "pt", "ps", "pr", "pq", "po", "pn", "pm", "pl", "pk", "pj", "pi", "ph", "pg", "pf", "pe", "pd", "pc", "pb", "pa", "oz", "oy", "ox", "ow", "ov", "ou", "ot", "os", "or", "oq", "op", "on", "om", "ol", "ok", "oj", "oi", "oh", "og", "of", "oe", "od", "oc", "ob", "oa", "nz", "ny", "nx", "nw", "nv", "nu", "nt", "ns", "nr", "nq", "np", "no", "nm", "nl", "nk", "nj", "ni", "nh", "ng", "nf", "ne", "nd", "nc", "nb", "na", "mz", "my", "mx", "mw", "mv", "mu", "mt", "ms", "mr", "mq", "mp", "mo", "mn", "ml", "mk", "mj", "mi", "mh", "mg", "mf", "me", "md", "mc", "mb", "ma", "lz", "ly", "lx", "lw", "lv", "lu", "lt", "ls", "lr", "lq", "lp", "lo", "ln", "lm", "lk", "lj", "li", "lh", "lg", "lf", "le", "ld", "lc", "lb", "la", "kz", "ky", "kx", "kw", "kv", "ku", "kt", "ks", "kr", "kq", "kp", "ko", "kn", "km", "kl", "kj", "ki", "kh", "kg", "kf", "ke", "kd", "kc", "kb", "ka", "jz", "jy", "jx", "jw", "jv", "ju", "jt", "js", "jr", "jq", "jp", "jo", "jn", "jm", "jl", "jk", "ji", "jh", "jg", "jf", "je", "jd", "jc", "jb", "ja", "iz", "iy", "ix", "iw", "iv", "iu", "it", "is", "ir", "iq", "ip", "io", "in", "im", "il", "ik", "ij", "ih", "ig", "if", "ie", "id", "ic", "ib", "ia", "hz", "hy", "hx", "hw", "hv", "hu", "ht", "hs", "hr", "hq", "hp", "ho", "hn", "hm", "hl", "hk", "hj", "hi", "hg", "hf", "he", "hd", "hc", "hb", "ha", "gz", "gy", "gx", "gw", "gv", "gu", "gt", "gs", "gr", "gq", "gp", "go", "gn", "gm", "gl", "gk", "gj", "gi", "gh", "gf", "ge", "gd", "gc", "gb", "ga", "fz", "fy", "fx", "fw", "fv", "fu", "ft", "fs", "fr", "fq", "fp", "fo", "fn", "fm", "fl", "fk", "fj", "fi", "fh", "fg", "fe", "fd", "fc", "fb", "fa", "ez", "ey", "ex", "ew", "ev", "eu", "et", "es", "er", "eq", "ep", "eo", "en", "em", "el", "ek", "ej", "ei", "eh", "eg", "ef", "ed", "ec", "eb", "ea", "dz", "dy", "dx", "dw", "dv", "du", "dt", "ds", "dr", "dq", "dp", "do", "dn", "dm", "dl", "dk", "dj", "di", "dh", "dg", "df", "de", "dc", "db", "da", "cz", "cy", "cx", "cw", "cv", "cu", "ct", "cs", "cr", "cq", "cp", "co", "cn", "cm", "cl", "ck", "cj", "ci", "ch", "cg", "cf", "ce", "cd", "cb", "ca", "bz", "by", "bx", "bw", "bv", "bu", "bt", "bs", "br", "bq", "bp", "bo", "bn", "bm", "bl", "bk", "bj", "bi", "bh", "bg", "bf", "be", "bd", "bc", "ba", "az", "ay", "ax", "aw", "av", "au", "at", "as", "ar", "aq", "ap", "ao", "an", "am", "al", "ak", "aj", "ai", "ah", "ag", "af", "ae", "ad", "ac", "ab", "z", "y", "x", "w", "v", "u", "t", "s", "r", "q", "p", "o", "n", "m", "l", "k", "j", "i", "h", "g", "f", "e", "d", "c", "b", "a"]
```

Comment:
Nice, I wanted to do something like this but I wasn't sure if we would run out of letters with just two characters. After using it for a bit I think it's safe to assume that it would be a crazy website if we run out of hints. We could always add more if we find use cases for that